lxc/image.go

```diff
@@ -288,14 +288,28 @@ func (c *imageCmd) run(config *lxd.Config, args []string) error {
 			inName = "default"
 		}
 
+		imageNames := []string{inName}
+
+		if len(args) > 2 {
+			for _, extraImage := range args[2:] {
+				imageNames = append(imageNames, extraImage)
+			}
+		}
+
 		d, err := lxd.NewClient(config, remote)
 		if err != nil {
 			return err
 		}
 
-		image := c.dereferenceAlias(d, inName)
-		err = d.DeleteImage(image)
-		return err
+		for _, imageName := range imageNames {
```

Comment:
Hmm, but then what happens if I do:

lxc remote delete host-a:image1 host-b:image2 host-c:image3

My point is that you should take all the args, then loop over them and do the whole thing for all of them as they may each have a different remote.